components/switch/blender/script.js

```diff
@@ -0,0 +1,13 @@
+$(function () {
+	$('[data-js="switch__version__"]').on('change', function () {
+		var $toggle = $(this).siblings('[data-js="switch-toggle__version__"]');
+		var className = $toggle.attr('class');
+		var classBits = className.split('-checked');
+
+		if (classBits.length > 1) {
+			$toggle.attr('class', classBits.join(''));
+		} else {
+			$toggle.attr('class', className.replace('toggle', 'toggle-checked'));
+		}
+	});
+});
```

Comment:
Sorry about this but I think we should make the switch no-js.
Same way I did it with the form-checks.
Just use css.
We found that using css selectors here actually verifies that the html state is what it is rather than using js in react to control it. It would still be a controlled input but the visual layer of it does not depend on state.